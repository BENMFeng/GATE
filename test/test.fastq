@HWI-ST1347:40:D18LNACXX:7:1101:14865:4731 1:N:0:ATNACG
ACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATTCAGATCGGAAGAGCACACGTCTGAACTCCAGTCACATCACGATCTCCTATGCCGTC
+
@@@DFFD8DCFFDFGGIJJIIJHDGG<C>G8?:?FGEGHGHBFHJIAHGGIB';?';BDEEEEB5?=7CC>>A3>@@C:AC>AACD?##############
@HWI-ST1347:40:D18LNACXX:7:1101:16901:21786 1:N:0:ATNACG
ACCACCGAGATCTACACTCTTTCCCTACACGACGCTATCTCGGTGGTAGATCGGAAGAGCACACGTCTGAACTCCAGTCACATCACGATCTCGTATGCCGT
+
?@@DDDDDH:DFHHIJJIJJICEHIIEGBAGGG<GGDDEHGEGFEGE>;@CE=B???;@36@CA:?;?@@:@C>A::>>:ACCCDD0222?@?2<?AC###
@HWI-ST1347:40:D18LNACXX:7:1101:18023:7205 1:N:0:ATNACG
GTAGATCGGAAGAGCACACGTCTGAACTCCAGTCACATCACGATCTCGTATGCCGTCTCATTAGATCGGAAGAGCACACGTCTGAACTCCAGTCACATCAC
+
B?@DFFFFHHHHGIJJJJJIGIJJHIIIJJIIHIDGHIIJJGFHIJJIFHIIHIIFIIIEHEACC>ABDFA>EB?@BCD@BBDDC@CCDDDC@CACCCCCC
@HWI-ST1350:31:C1427ACXX:6:1101:9783:2489 1:N:0:CAGATC
CACGAGCCATTGAGTCTCGCCTGAACATTCACTGCAATAAGGTCGCGCAGACGCGCCTTATTCCACTGCGTGAGCTTGAGCGTATCTGCAGGAAAGATCGG
+
CCCFFFFFHHHHHJHIJJJJJJJJJJJJJJJIJJIIJJJJJJGHIJJJJJJHHFDDDDDDDEEEEDDDDDBDDDCDDDDDDD@DDDEEDDDDBDDDCDDDD
@test
CACGAGCCATTGAGTCTCGCCTGAACAAGATCTCGGTGGTCGCCGTATCATTCAGATCCAAGCAGAAGACGGCATACGAGATGCGTGAGCTTGAGCGAAAA
+
CCCFFFFFHHHHHJHIJJJJJJJJJJJJJJJIJJIIJJJJJJGHIJJJJJJHHFDDDDDDDEEEEDDDDDBDDDCDDDDDDD@DDDEEDDDDBDDDCDDDD
