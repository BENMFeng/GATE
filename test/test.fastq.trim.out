@HWI-ST1350:31:C1427ACXX:6:1101:9783:2489 1:N:0:CAGATC Multiplexing_Index_Read_Sequencing_Primer_5' 3'_trimmed: 7
CACGAGCCATTGAGTCTCGCCTGAACATTCACTGCAATAAGGTCGCGCAGACGCGCCTTATTCCACTGCGTGAGCTTGAGCGTATCTGCAGGAA
+
CCCFFFFFHHHHHJHIJJJJJJJJJJJJJJJIJJIIJJJJJJGHIJJJJJJHHFDDDDDDDEEEEDDDDDBDDDCDDDDDDD@DDDEEDDDDBD
